packages/rust-bindings/src/aux_functions.rs

```diff
@@ -97,6 +188,7 @@ pub fn generate_unsigned_credential_aux(input: &str) -> Result<String> {
         policy,
         cred_key_info,
         address.as_ref(),
+        &SystemAttributeRandomness {},
```

Comment:
Did the interface of `create_unsigned_credential` change so that this change is necessary?

Reply:
Yes, it was changed to allow the randomness to be chosen. This just means that the function will use whatever the system offers. (Which is what it did before)